_breadcrumbs.scss

```diff
@@ -0,0 +1,116 @@
+// Breadcrumbs
+// ===
+
+
+// Configurable Variables
+// ---
+
+$breadcrumbs__divider-content: '>' !default; // TO DO: include stencil icon
+$breadcrumbs__show-more-content: '...' !default; // TO DO: include stencil icon
+
+
+// Dependencies
+// ---
+
+@import 'stencil-variables/variables';
+@import 'spline/dist/utilities/hide-text';
+@import 'spline/dist/utilities/icon-font-replace';
+@import 'level/level';
+
+
+// Breadcrumbs
+// ---
+//
+
+.c-breadcrumbs {
+    display: flex;
+    overflow: auto;
+    align-items: center;
+    margin: 0;
+    padding: 0;
+
+    list-style: none;
+
+    -webkit-overflow-scrolling: touch;
```

Comment:
I’m not sure if this is a good idea or a terrible idea, but what about suppressing scrollbars here, and relying on some other visual cue that the content is cut off? Not sure if this still works in iOS, but could do it with: 
```
.c-breadcrumbs::-webkit-scrollbar {
    display: none;
}
```